openassessment/xblock/static/js/src/studio/oa_edit_prompts.js

```diff
@@ -21,7 +23,10 @@ OpenAssessment.EditPromptsView = function(element, notifier) {
             removeButtonClass: "openassessment_prompt_remove_button",
             containerItemClass: "openassessment_prompt",
             notifier: notifier,
-            addRemoveEnabled: this.addRemoveEnabled
+            addRemoveEnabled: this.addRemoveEnabled,
```

Comment:
Why did you add "addRemoveEnabled"?

Reply:
@sandroroux it wasn't me :wink:  I've just added comma - i may assume it is the reason of this misunderstanding :smiley:

Reply:
Ah, my mistake. Thanks for clarifying!